Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.10
```

Python 3.11 seems to work fine as well, for that extra performance. 🚀 
```suggestion
FROM python:3.11
```